contracts/staking/PoSValidatorManager.sol

```diff
@@ -56,6 +56,12 @@ abstract contract PoSValidatorManager is
         mapping(bytes32 validationID => PoSValidatorRequirements) _validatorRequirements;
         /// @notice Maps the delegationID to the delegator information.
         mapping(bytes32 delegationID => Delegator) _delegatorStakes;
+        /// @notice Maps the delegationID to its pending staking rewards.
+        mapping(bytes32 delegationID => uint256) _redeemableDelegatorRewards;
+        /// @notice Maps the validator owner address to its pending staking rewards.
+        mapping(bytes32 validationID => uint256) _redeemableValidatorRewards;
+        /// @notice Saves the uptime of a pending completed or completed validation period so that delegators can collect rewards.
+        mapping(bytes32 validationID => uint64) _completedValidationUptimeSeconds;
```

Comment:
Are we able to clean up the entries in this map when the final delegation to a completed validation is also completed? Not sure if we have a counter on the number of delegations for a given validation or not as is.